content/applications/general/email_communication/email_common.rst

```diff
@@ -0,0 +1,123 @@
+
+Why is my email sent late?
+**************************
+
+It may happen that you schedule a email campaign but it is not sent on time. You should know that we use a delayed job to send emails that we consider as not urgent (Newsletters concept such as mass mailing, marketing automation, events). The system utility **cron** can be used to schedule programs to run automatically at predetermined intervals. We use that policy in order to avoid cluttering the mail servers and prioritize the communication. 
+
+The emails considered as urgent (communication from a person to another one such as Sales Order, Invoices, Purchase Orders…) are sent directly. 
+
+.. image:: email_common/email_scheduled_later.png
+   :align: center
+   :alt: Email scheduled to be sent later.
+
+By default, the Mass Mailing cron runs every 60 minutes. So, you should wait an hour before the campaign is actually sent.
+
+Incoming emails
+===============
+
+When you have an issue with incoming emails, there might not be an indication per se in Odoo. This is the client who tries to contact a database who will get a bounce (most of the time 550: mailbox unavailable).
+
+Emails are not received
+-----------------------
+
+Depending on the platform you are using: 
+ - The **Odoo.sh** users can find their live logs on the folder ~/logs/.
+ - The folder ~/logs/ (preferably accessed by the command line) of an Odoo.sh contains a list of files containing the logs of the database. The log files are created everyday at 4:00 AM UTC. The two last days are not compressed, while the older ones are, in order to gain space. The naming of the files are decremental, which means that the file named .1 is the one from yesterday and the file .2 is the day before yesterday. See the following Odoo.sh documentation about `logs <https://www.odoo.com/documentation/master/administration/odoo_sh/getting_started/branches.html?#logs>`_. Use the command ``grep`` and ``zgrep`` (for the compressed ones) to search through the files. 
+ - The **SaaS** users won’t have access to their logs. However you can still contact `Odoo Support <https://www.odoo.com/help>`_ , if you have a recurring issue with the same client or domain.
+
+Get helped from support
+-----------------------
+
+In order to get helped efficiently, please provide as much information as possible. Here is a list of what can be helpful:
+ - The **EML** of the email, stating for *Electronic Mail*, is the file format containing all the technical information required for an investigation. The documentation of your own email provider might help you on how to get your EML files. Once you get the EML of the email, adding it in the attachment of your ticket is the most efficient way for us to investigate. The support will mainly focus on redundant issues.
+
+
+.. seealso:: Here are links to the documentation of some common providers:
```

Comment:
Either move this after the bulleted list or indent it to make it part of the bullet point.